README.md

```diff
@@ -1 +1,44 @@
-# Portfolio1
\ No newline at end of file
+# Portfolio
+
+> This is my first attempt at making my portfolio. 
+

```

Comment:
- [ ] **OPTIONAL**
It's always better when you attach a screenshot of your project in the README file. It helps to make it professional and good-looking.
_You can always refer to this [template](https://github.com/microverseinc/readme-template) as a reminder._